src/components/PostList/index.module.scss

```diff
@@ -51,22 +39,11 @@
   line-height: 28px;
 
   @include media(">=desktop") {
-    margin-bottom: 28px;
-
+    margin-bottom: 40px;
     font-size: 28px;
     line-height: 40px;
   }
 
-  @include media("grid-9") {
-    grid-column: 1 / span 6;
-    grid-row: 1 / span 1;
-    margin-bottom: 0;
-  }
-
-  @include media("grid-10") {
-    grid-column-start: 2;
-  }
-
   /**
    * Fix for Firefox font rendering issues.
    */
```

Comment:
Is this still needed?

Reply:
I guess? I'm not sure if something has changed since this was added.